schema/cassandra/temporal/schema.cql

```diff
@@ -97,17 +97,21 @@ CREATE TABLE tasks (
   };
 
 -- Stores task queue information such as user provided versioning data
+-- OR
+-- Used as a mapping from build id to task queue
 CREATE TABLE task_queue_user_data (
-  namespace_id    uuid,
-  task_queue_name text,
-  data            blob,   -- temporal.server.api.persistence.v1.TaskQueueUserData
-  data_encoding   text,   -- Encoding type used for serialization, in practice this should always be proto3
-  version         bigint, -- Version of this row, used for optimistic concurrency
+  namespace_id                uuid,
+  task_queue_name             text,
+  build_id_if_row_is_an_index text,      -- If this row is used as a mapping of build id to task queue, this will not be empty
```

Comment:
I like the explicit and long name TBH.

Reply:
what if you want to define another "index" by a different field, so now the name is wrong because there are two different types of index?

also this is nitpicky but, the "row" is not "an index", it's an entry in an index. so the name isn't even really correct